web/themes/custom/pineapple/templates/dataset/table--edit-password-policy-status.html.twig

```diff
@@ -0,0 +1,115 @@
+{#
```

Comment:
Delete this file. There's no reason to not make every table on your site responsive.

Reply:
Is what I've done okay?

Reply:
Yes